app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
```

Comment:
You can iterate through the list, without construction `range(len(friends))`. Also, it would be grate to change variable name to more informative, smth like 'friend', so finally it will looks like `for friend in friends`